ihmc-high-level-behaviors/src/libgdx/java/us/ihmc/rdx/behaviorTree/RDXCRDTTools.java

```diff
@@ -46,4 +48,20 @@ else if (latestTimestampModifiable.isModified())
 
       gizmo.update();
    }
+
+   public static void toLibGDX(RigidBodyTransformMessage rigidBodyTransformMessage, Matrix4 gdxAffineToPack)
```

Comment:
No where else has both these on the classpath, so i put this here, its convenient